api/v1alpha1/accesslogging_types.go

```diff
@@ -96,11 +96,14 @@ type FileEnvoyProxyAccessLog struct {
 
 // OpenTelemetryEnvoyProxyAccessLog defines the OpenTelemetry access log sink.
 //
-// +kubebuilder:validation:XValidation:message="BackendRef only support Service Kind.",rule="!has(self.backendRef) || !has(self.backendRef.kind) || self.backendRef.kind == 'Service'"
+// +kubebuilder:validation:XValidation:message="host or backendRef needs to be set",rule="has(self.host) || has(self.backendRef)"
+// +kubebuilder:validation:XValidation:message="backendRef only support Service Kind.",rule="!has(self.backendRef) || self.backendRef.kind == 'Service'"
```

Comment:
TBH, the previous version was more rigorous, but fails on `1.28.0`.
Good news, there's dafault value on `kind`.